.github/scripts/process-attribution.sh

```
@@ -0,0 +1,258 @@
+fi
+
+if [[ -f "$container_io_state_file" ]]; then
+  while IFS='|' read -r container_id read_bytes write_bytes read_ops write_ops; do
+    [[ -n "$container_id" ]] && previous_container_io["$container_id"]="$read_bytes|$write_bytes|$read_ops|$write_ops"
+  done < "$container_io_state_file"
+fi
+
+read_process_label() {
+  local command_line="$1"
+  case "$command_line" in
+    *"bulldozer"*|*"start:bulldozer"*) printf '%s' "bulldozer" ;;
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 57.

(SC2221)